demo/common/src/lib.rs

```diff
@@ -923,19 +914,48 @@ fn center_of_window(window_size: &WindowSize) -> Point2DF32 {
 
 enum Camera {
     TwoD(Transform2DF32),
-    ThreeD { transform: CameraTransform3D, velocity: Point3DF32 },
+    ThreeD {
+        // The mode will either be 3D or VR
+        mode: Mode,
```

Comment:
This seems redundant; the mode can be detected by looking at the length of `transforms`. How about making this a method (e.g. `is_stereo()`) on `Camera`?

Reply:
Fixed. I kept the `mode()` method, rather than going for `is_stereo()`, just because that fits the way the method is used.